presto-main/src/test/java/io/prestosql/operator/spiller/BenchmarkBinaryFileSpiller.java

```diff
@@ -55,8 +55,8 @@
 
 @State(Scope.Thread)
 @OutputTimeUnit(SECONDS)
-@Fork(3)
-@Warmup(iterations = 5, time = 500, timeUnit = TimeUnit.MILLISECONDS)
+@Fork(1)
+@Warmup(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
```

Comment:
are changes here intentional?

Reply:
Woops. Definitely not. Will back those out.

Reply:
Done.